patterns/jolla-configuration-h3413.yaml

```diff
@@ -17,13 +17,17 @@ Requires:
 
 - sailfish-content-graphics-z@ICON_RES@
 
-- jolla-settings-system-nfc
 - jolla-settings-system-flashlight
 - jolla-settings-networking-multisim
```

Comment:
Hmm, this multisim should not be there...

Reply:
Will take it out